dask/dataframe/io/tests/test_parquet.py

```diff
@@ -597,3 +596,21 @@ def test_parquet_select_cats(tmpdir):
     assert list(rddf.columns) == ['ints']
     rddf = dd.read_parquet(fn)
     assert list(rddf.columns) == list(df)
+
+
+@pytest.mark.parametrize('compression,', ['default', None, 'gzip'])
+def test_writing_parquet_with_compression(tmpdir, compression):
+    fn = str(tmpdir)
+
+    df = pd.DataFrame({'x': ['a', 'b', 'c'] * 10, 'y': [1, 2, 3] * 10})
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    check_fastparquet()
+    ddf.to_parquet(fn, compression=compression, engine='fastparquet')
+    out = dd.read_parquet(fn, engine='fastparquet')
+    assert_eq(out, ddf)
```

Comment:
It would be better if you used the `engine` fixture as I posted before. As is, both the fastparquet and `arrow` test uses the same temporary directory, and the arrow tests will only run if fastparquet is installed (since that's checked first).

```python
def test_writing_parquet_with_compression(tmpdir, compression, engine):
   ...
```